dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -16,6 +17,7 @@
 import io.swagger.model.ToolDescriptor;
 import io.swagger.model.ToolDockerfile;
 import io.swagger.model.ToolVersion;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Not really too important, but I believe there's an org.apache.commons.lang3.StringUtils.  It might remove the need to add an extra dependency.